src/lib/logger.js

```
@@ -0,0 +1,161 @@
+      if (data) {
+        console.info(this.formatMessage(LOG_LEVELS.INFO, message), data);
+      } else {
+        console.info(this.formatMessage(LOG_LEVELS.INFO, message));
+      }
+    }
+  }
+
+  debug(message, data = null) {
+    if (this.shouldLog(LOG_LEVELS.DEBUG)) {
+      if (data) {
+        console.log(this.formatMessage(LOG_LEVELS.DEBUG, message), data);
```

ESLint

[error] Use logger from @/lib/logger instead of console.log

(no-restricted-syntax)